e2e/test/scenarios/embedding-sdk/interactive-question.cy.spec.ts

```diff
@@ -107,39 +94,51 @@ describeSDK("scenarios > embedding-sdk > interactive-question", () => {
       expect(response?.statusCode).to.equal(202);
     });
 
-    tableInteractive().findByText("Max of Quantity").should("be.visible");
+    tableInteractive().findByText("Product ID").should("be.visible");
 
-    tableHeaderClick("Max of Quantity");
+    tableHeaderClick("Product ID");
 
-    popover()
-      .findByTestId("click-actions-sort-control-formatting-hide")
-      .click();
+    popover().within(() => {
+      cy.findByTestId("click-actions-sort-control-formatting-hide").click();
+    });
 
-    tableInteractive().findByText("Max of Quantity").should("not.exist");
+    tableInteractive().findByText("Product ID").should("not.exist");
   });
 
-  it("can save a question", () => {
-    cy.intercept("POST", "/api/card").as("createCard");
-
-    cy.findAllByTestId("cell-data").last().click();
-
-    popover().findByText("See these Orders").click();
+  it("can save a question to a default collection", () => {
+    saveInteractiveQuestionAsNewQuestion("Sample Orders 1");
 
-    cy.findByRole("button", { name: "Save" }).click();
+    cy.wait("@createCard").then(({ response }) => {
+      expect(response?.statusCode).to.equal(200);
+      expect(response?.body.name).to.equal("Sample Orders 1");
+      expect(response?.body.collection_id).to.equal(null);
```

Comment:
Ah, that makes sense! My guess is `null` defaults to a sensible default collection for a user? (I could be wrong)

Reply:
If a user doesn't have access to the root collection then how can they create a question in the root collection?

AFAIK, we have two ways to represent the root collection one is `"root"` and another one is `null`.

![image](https://github.com/user-attachments/assets/42131464-9e48-4d42-95f6-07f9762ad010)

Reply:
@WiNloSt This is unrelated to this PR, but I just wonder if users without root collection access will get a sensible default collection in the collection picker that they have write access to, or does it always default to the root collection. My guess it that they can see that the collection exist, but can't save to it.

Reply:
From my testing, users would be led to the first available collection. Please note that the permission isn't permissive, even if the `root` collection are inaccessible users would still be able to access nested collections if they have permissions for those collections.

Otherwise if all collections are inaccessible, they can only access their personal collection.

Reply:
Oh nice, that's the ideal behaviour then!